ts/features/bonus/bpd/screens/details/components/summary/BpdSummaryComponent.tsx

```diff
@@ -0,0 +1,85 @@
+import { fromNullable } from "fp-ts/lib/Option";
+import * as pot from "italia-ts-commons/lib/pot";
+import { View } from "native-base";
+import * as React from "react";
+import { connect } from "react-redux";
+import { Dispatch } from "redux";
+import { profileNameSelector } from "../../../../../../../store/reducers/profile";
+import { GlobalState } from "../../../../../../../store/reducers/types";
+import { isStringNullyOrEmpty } from "../../../../../../../utils/strings";
+import { isReady } from "../../../../model/RemoteValue";
+import { BpdAmount } from "../../../../store/actions/amount";
+import { BpdPeriod } from "../../../../store/actions/periods";
+import { bpdIbanSelector } from "../../../../store/reducers/details/activation";
+import { bpdAmountForSelectedPeriod } from "../../../../store/reducers/details/amounts";
+import { bpdSelectedPeriodSelector } from "../../../../store/reducers/details/selectedPeriod";
+import { TextualSummary } from "./TextualSummary";
+import { TransactionsGraphicalSummary } from "./TransactionsGraphicalSummary";
+
+type Props = ReturnType<typeof mapDispatchToProps> &
+  ReturnType<typeof mapStateToProps>;
+
+type SummaryData = {
+  period: BpdPeriod;
+  amount: BpdAmount;
+  name: string | undefined;
+};
+
+const Content = (sd: SummaryData) => (
+  <View>
+    {sd.period.status === "Inactive" ? null : (
+      <TransactionsGraphicalSummary
+        transactions={sd.amount.transactionNumber}
+        minTransactions={sd.period.minTransactionNumber}
+      />
+    )}
+    <View spacer={true} />
+    <TextualSummary period={sd.period} amount={sd.amount} name={sd.name} />
```

Comment:
```suggestion
    <TextualSummary {...sd} />
```
Surely more concise but less readable
Wht's your opinion?

Reply:
Sure better!